config.json

```diff
@@ -1,4 +1,4 @@
 {
     "CAIRO_LANG": "0.9.1",
-    "STARKNET_DEVNET": "0.2.6"
+    "STARKNET_DEVNET": "0.2.7"
```

Comment:
In the meantime, devnet's been upgraded to 0.2.8, you can change that as well.

Reply:
https://github.com/Shard-Labs/starknet-hardhat-plugin/pull/179/commits/addd7103abc3965d33c5e63467d759ccd53d3265